releaseNotes/JavaScriptProvider.md

```diff
@@ -2,6 +2,24 @@
 
 [Source code][source_code] | [Package (npm)][package] | [Samples][samples]
 
+## 1.0.0 - June 6, 2024
+
+This is the first stable release of the following features.
+- Loading and composing configuration with key-value selectors.
+- Authenticating with connection string or Microsoft Entra Id.
+- Loading key-values as either a `Map` or an object based on needs.
+- Trimming prefixes from keys.
+- Loading key-values with JSON content type direct as an object.
+- Resolving Key Vault reference.

```

Comment:
"Resolving Key Vault reference**s**" or "Key Vault reference resolution"

Reply:
updated to "Key Vault reference resolution"